rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout, .sibbo-panel"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
+  "optIn": [{ "click": "#acceptAllMain" }],
+  "optOut": [
+    { "click": ".sibbo-panel__aside__buttons a[data-nav=\"purposes\"]" },
+    { "click": ".sibbo-panel__main__header__actions a[data-focusable=\"reject-all\"]" },
+    { "wait": "500" },
+    { "click": ".sibbo-panel__main__footer__actions a[data-focusable=\"next\"]" },
+    { "click": ".sibbo-panel__main div[data-view=\"purposesLegInt\"] a[data-focusable=\"reject-all\"]" },
+    { "wait": "500" },
+    { "click": "#purposesLegIntSaveAndExit" }
+  ],
+  "test": [
+    {
+      "eval": "!document.cookie.includes('_gpi') && !document.cookie.includes('_gads')"
```

Comment:
judging purely by the names, these cookies might be set by google, not by the CMP itself. How about checking if `euconsent-v2` is set?

Reply:
Eval replaced by the following (L53): `"eval": "window.localStorage.getItem('euconsent-v2')"`